source/common/runtime/runtime_features.cc

```diff
@@ -179,6 +179,7 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_use_network_type_socket_option);
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_quic_disable_client_early_data);
 
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_ext_proc_graceful_grpc_close);
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_ext_proc_modified_append_default_value);
```

Comment:
Changing the runtime default value to false based on an offline discussion about keeping the legacy behavior for now. This enables us to collect the data about how many ext_proc servers out there are not explicitly encoding "append".